gui/src/components/mainInput/TipTapEditor.tsx

```diff
@@ -854,61 +1007,6 @@ const TipTapEditor = memo(function TipTapEditor({
 
   const [optionKeyHeld, setOptionKeyHeld] = useState(false);
 
-  // Use onTransaction to track content changes
```

Comment:
This section of code was no longer needed as it is being handled elsewhere in TipTapEditor